File: Bars.GkhRF/libs/B4/model/ContractRF.js
Ext.define('B4.model.ContractRf', {
    extend: 'B4.base.Model',
    idProperty: 'Id',
    proxy: {
        type: 'b4proxy',
        controllerName: 'ContractRf'
    },
    fields: [
        { name: 'Id', useNull: true },
        { name: 'ManagingOrganization', defaultValue: null },
        { name: 'ContragentName' },
        { name: 'DocumentNum' },
        { name: 'DocumentDate', defaultValue: null },
        { name: 'DateBegin', defaultValue: null },
        { name: 'DateEnd', defaultValue: null },
        { name: 'MunicipalityName' },
        { name: 'ManagingOrganizationName' },
        { name: 'ContractRfObjectsCount' },
        { name: 'File', defaultValue: null },
        { name: 'SumAreaMkd' },
        { name: 'TerminationContractNum' },
        { name: 'TerminationContractDate' },
        { name: 'TerminationContractFile', defaultValue: null },
        { name: 'SumAreaLivingOwned' }
    ]
});

File: Bars.GkhRF/Entities/Contract/ContractRF.cs
namespace Bars.GkhRf.Entities
{
    using System;

    using Bars.B4.Modules.FileStorage;
    using Bars.Gkh.Entities;

    /// <summary>
    /// Договор рег. фонда
    /// </summary>
    public class ContractRf : BaseGkhEntity
    {
        /// <summary>
        /// Управляющая организация
        /// </summary>
        public virtual ManagingOrganization ManagingOrganization { get; set; }

        /// <summary>
        /// Номер документа
        /// </summary>
        public virtual string DocumentNum { get; set; }

        /// <summary>
        /// Дата документа
        /// </summary>
        public virtual DateTime? DocumentDate { get; set; }

        /// <summary>
        /// Дата начала действия
        /// </summary>
        public virtual DateTime? DateBegin { get; set; }

        /// <summary>
        /// Дата окончания действия
        /// </summary>
        public virtual DateTime? DateEnd { get; set; }

        /// <summary>
        /// Файл
        /// </summary>
        public virtual FileInfo File { get; set; }

        /// <summary>
        /// Номер прекращения контракта
        /// </summary>
        public virtual string TerminationContractNum { get; set; }

        /// <summary>
        /// Дата прекращения контракта
        /// </summary>
        public virtual DateTime? TerminationContractDate { get; set; }

        /// <summary>
        /// Файл прекращения контракта
        /// </summary>
        public virtual FileInfo TerminationContractFile { get; set; }
    }
}